src/sentry/api/validators/doc_integration.py

```diff
@@ -0,0 +1,30 @@
+import logging
+
+from jsonschema import Draft7Validator
+from jsonschema.exceptions import best_match
+
+from sentry.utils.json import JSONData
+
+logger = logging.getLogger(__name__)
+
+METADATA_SCHEMA = {
+    "type": "object",
+    "definitions": {
+        "resource": {
+            "type": "object",
+            "properties": {"title": {"type": "string"}, "url": {"type": "string", "format": "uri"}},
+            "required": ["title", "url"],
+            "additionalProperties": False,
+        }
+    },
+    "properties": {"resources": {"type": "array", "items": {"$ref": "#/definitions/resource"}}},
+    "additionalProperties": False,
+}
+
+METADATA_PROPERTIES = list(METADATA_SCHEMA["properties"].keys())
+
+
+def validate_metadata_schema(instance: JSONData):
```

Comment:
So no necessary but what I like to do with validators is cast the parameter to a type and return it when it is valid. For example:
```py
issued_date: datetime = validate_datetime(request.body["issued_date"])
```

Reply:
done!